packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -33,6 +33,39 @@ class JavascriptMessage {
 /// Callback type for handling messages sent from Javascript running in a web view.
 typedef void JavascriptMessageHandler(JavascriptMessage message);
 
+/// Information about a navigation action that is about to be executed.
+class NavigationRequest {
+  NavigationRequest({this.url, this.isMainFrame});
```

Comment:
constructor is missing docs

Reply:
consider adding some sort of benchmark to check how many public members are missing docs in the repo

Reply:
Made the constructor private, filed: https://github.com/flutter/flutter/issues/28599